dlp/src/test/java/com/example/dlp/InspectIT.java

```diff
@@ -44,7 +44,7 @@
   public void setUp() {
     bout = new ByteArrayOutputStream();
     out = new PrintStream(bout);
-    System.setOut(out);
+    System.setOut(out); // TODO(b/64541432) DLP currently doesn't support GOOGLE DEFAULT AUTH
```

Comment:
Les, can we refer to the [Github issue](https://github.com/GoogleCloudPlatform/google-cloud-java/issues/2231) instead : so users can view details ?